includes/orders/functions/actions.php

```diff
@@ -151,22 +151,24 @@ function edd_add_manual_order( $args = array() ) {
 	$order_total    = floatval( $data['total'] );
 
 	// Add the order ID
-	$order_id = edd_add_order( array(
-		'status'         => 'pending', // Always insert as pending initially.
-		'user_id'        => $user_id,
-		'customer_id'    => $customer_id,
-		'email'          => $email,
-		'ip'             => sanitize_text_field( $data['ip'] ),
-		'gateway'        => sanitize_text_field( $data['gateway'] ),
-		'mode'           => $mode,
-		'currency'       => edd_get_currency(),
-		'payment_key'    => sanitize_text_field( $data['payment_key'] ),
-		'subtotal'       => $order_subtotal,
-		'tax'            => $order_tax,
-		'discount'       => $order_discount,
-		'total'          => $order_total,
-		'date_created'   => $date,
-	) );
+	$order_id = edd_add_order(
+		array(
+			'status'       => 'pending', // Always insert as pending initially.
+			'user_id'      => $user_id,
+			'customer_id'  => $customer_id,
+			'email'        => $email,
+			'ip'           => sanitize_text_field( $data['ip'] ),
+			'gateway'      => sanitize_text_field( $data['gateway'] ),
+			'mode'         => $mode,
+			'currency'     => edd_get_currency(),
+			'payment_key'  => edd_generate_order_payment_key( $email ),
```

Comment:
@LisaCee The function is working great to generate a key. However when adding an Order in the admin we have the `Key` field available to enter a custom one:

<img width="299" alt="Screen Shot 2020-06-11 at 2 18 06 PM" src="https://user-images.githubusercontent.com/491124/84424641-66979e00-abee-11ea-9d04-66ad0d351786.png">
 
We'll need to check if `$data['payment_key']` is empty and only use a generated key if it is -- otherwise use the input value stored in `$data['payment_key']` that needs to be sanitized via `sanitize_text_field()`